src/main/index.js

```diff
@@ -6,12 +6,20 @@ import { migrateSettings } from './migrate-settings';
 import { initiateAutoUpdater } from './auto-updater';
 import tray from './tray';
 import getMenu from './get-menu';
-import installExtension, { REACT_DEVELOPER_TOOLS, REDUX_DEVTOOLS } from 'electron-devtools-installer';
 
 const settings = require('electron-settings');
-
 require('electron-dl')({saveAs: true});
 
+const installExtensions = async () => {
+  const installer = require('electron-devtools-installer');
+  const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
```

Comment:
Do the developer needs to set the env variable by himself? Or is it some automagic that will set this?

Reply:
The developer needs to set the env only if he wants to force updates [see DevTools](https://github.com/electron-react-boilerplate/electron-react-boilerplate/wiki/DevTools).

```
$ UPGRADE_EXTENSIONS=1 yarn run dev
# For Windows
$ set UPGRADE_EXTENSIONS=1 && yarn run dev```